qanta/wikipedia/wikification.py

```diff
@@ -12,6 +12,7 @@ def main():
     parser.add_argument("--output_directory", type=str,
                         default="output/wikifier/data/input/",
                         help="Where we write output file")
+    parser.add_argument("--database", type=int, default=1, help="1 to load questions, 0 for no")
```

Comment:
I'm not sure this is a right thing to add.  But it allowed for the loading of entities from the local db file.